README.md

```diff
@@ -88,30 +79,53 @@ callback:
 
 ```js
 // Create a new customer and then a new charge for that customer:
-stripe.customers.create({
-  email: 'foo-customer@example.com'
-}).then((customer) => {
-  return stripe.customers.createSource(customer.id, {
-    source: 'tok_visa'
-  });
-}).then((source) => {
-  return stripe.charges.create({
-    amount: 1600,
-    currency: 'usd',
-    customer: source.customer
+stripe.customers
+  .create({
+    email: 'foo-customer@example.com',
+  })
+  .then((customer) => {
+    return stripe.customers.createSource(customer.id, {
+      source: 'tok_visa',
+    });
+  })
+  .then((source) => {
+    return stripe.charges.create({
+      amount: 1600,
+      currency: 'usd',
+      customer: source.customer,
+    });
+  })
+  .then((charge) => {
+    // New charge created on a new customer
+  })
+  .catch((err) => {
+    // Deal with an error
   });
-}).then((charge) => {
-  // New charge created on a new customer
-}).catch((err) => {
-  // Deal with an error
-});
+```
+
+### Using callbacks
+
+On versions of Node.js prior to v7.9:
+
+```js
+var stripe = require('stripe')('sk_test_...');
+
+stripe.customers.create(
+  {
+    email: 'customer@example.com',
+  },
+  function(err, customer) {
+    err; // null if no error occurred
```

Comment:
Nit: Can we do something with these values? It looks a little odd by themselves. Perhaps something like:

```
if (err) {
  // Deal with an error
}

// Do something with created customer object
console.log(customer.id) 
```